en/peripherals/remote_id.md

```diff
@@ -168,10 +201,48 @@ Some known issues are:
   - PX4 v1.14 does not yet receive `OPEN_DRONE_ID_ARM_STATUS`.
 - `OPEN_DRONE_ID_ARM_STATUS` must be forwarded to the GCS, if present for additional error reporting.
 - [OPEN_DRONE_ID_BASIC_ID](https://mavlink.io/en/messages/common.html#OPEN_DRONE_ID_BASIC_ID) specifies a serial number in an invalid format (not ANSI/CTA-2063 format).
-- The vehicle ID is expected to be tamper resistent.
+- The vehicle ID is expected to be tamper resistant.
 
 [PX4-Autopilot/21647](https://github.com/PX4/PX4-Autopilot/pull/21647) is intended to address the known issues.
 
+## Troubleshooting
+
+### CAN Remote ID Not Working
+
+::: info
+This information was tested with the CAN Cube ID from CubePilot.
+It _should_ also apply to CAN Remote ID modules from other vendors.
+:::
+
+To confirm that the Remote ID is working:
+
+- Check that the Remote_ID node appears on the UAVCAN list.
+
+  Run the following command in the [QGroundControl MAVLink Console](http://localhost:5173/px4_user_guide/en/debug/mavlink_shell.html#qgroundcontrol-mavlink-console):
+
+  ```sh
+  uavcan status
+  ```
+
+  The remote id should appears in the list as shown below (there maybe other nodes in the list too).
+
+  ```plain
+  Online nodes (Node ID, Health, Mode):
+     125 OK         OPERAT
```

Comment:
Is this really what you see? Does that mean we're looking for a node with id 125? I.e. how do you know this node is a remote ID node - in particular if there are multiples?

Reply:
So Enter UAVCAN generates a lot of text that I didn't want to put all of it making the document too long. When only the Cube ID is attached then yes this is the only node that is present but there can be other nodes attached.

Reply:
That's cool, but it doesn't quite confirm what I need. Let me put it another way - is 125 always the node of any remote id? 

Otherwise, i I think you are saying that there is no way to tell if a remote id is present unless you diff what the list looks like before and after.

Reply:
The safest approach would be to diff between the before and after on the CAN bus. Not sure if all Remote ID modules will have 125 as there Node ID. I was only able to test out the Cube ID.

Reply:
Thanks @Aaron1356 . I have edited this to put the message check first - if that works you don't need to do anything. Then for the `uavcan status` check I have suggested a simple node count as first option, and differencing as a better check. I think this is OK.